index.js

```diff
@@ -516,129 +518,59 @@ function Cardboard(config) {
-        // we split it.
-        else if (bbox[0] <= 180 && bbox[2] >= 180) {
-            bboxes = bboxes.reduce(function(memo, bbox) {
-                memo.push([bbox[0], bbox[1], 180 - epsilon, bbox[3]]);
-                memo.push([180 + epsilon, bbox[1], bbox[2], bbox[3]]);
-                return memo;
-            }, []);
-
-            if (bbox[1] <= 0 && bbox[3] >= 0) {
-                bboxes = bboxes.reduce(function(memo, bbox) {
-                    memo.push([bbox[0], bbox[1], bbox[2], -epsilon]);
-                    memo.push([bbox[0], epsilon, bbox[2], bbox[3]]);
-                    return memo;
-                }, []);
-            }
+    cardboard.bboxQuery = function(bbox, dataset, options, callback) {
+        if (typeof options === 'function') {
+            callback = options;
+            options = {};
         }
 
-        // If a query crosses the equator/prime meridian, we split it.
-        else if (bbox[0] <= 0 && bbox[2] >= 0) {
-            bboxes = bboxes.reduce(function(memo, bbox) {
-                memo.push([bbox[0], bbox[1], -epsilon, bbox[3]]);
-                memo.push([epsilon, bbox[1], bbox[2], bbox[3]]);
-                return memo;
-            }, []);
-
-            if (bbox[1] <= 0 && bbox[3] >= 0) {
-                bboxes = bboxes.reduce(function(memo, bbox) {
-                    memo.push([bbox[0], bbox[1], bbox[2], -epsilon]);
-                    memo.push([bbox[0], epsilon, bbox[2], bbox[3]]);
-                    return memo;
-                }, []);
-            }
-        }
-
-        var tiles = bboxes.map(function(bbox) {
-            return tilebelt.bboxToTile(bbox);
-        });
-
-        // Deduplicate subquery tiles.
-        uniq(tiles, function(a, b) {
-            return !tilebelt.tilesEqual(a, b);
-        });
-
-        if (tiles.length > 1) {
-            // Filter out the z0 tile -- we'll always search it eventually.
-            tiles = _.filter(tiles, function(item) {
-                return item[2] !== 0;
-            });
-        }
-
-        tiles.forEach(function(tile) {
-            var tileKey = tilebelt.tileToQuadkey(tile);
-
-            // First find features indexed in children of this tile
-            var query = {
-                cell: { BEGINS_WITH: 'cell!' + tileKey },
-                dataset: { EQ: dataset }
-            };
-
-            var options = {
-                pages: 0,
-                index: 'cell',
-                filter: {
-                    west: { LE: bbox[2] },
-                    east: { GE: bbox[0] },
-                    north: { GE: bbox[1] },
-                    south: { LE: bbox[3] }
-                }
-            };
-            q.defer(config.dyno.query, query, options);
+        options = _.defaults(options, { limit: 100 });
 
-            // Travel up the parent tiles, finding features indexed in each
-            var parentTileKey = tileKey.slice(0, -1);
+        // First find features indexed in children of this tile
```

Comment:
Trivial, but this comment confused me for a minute since we're not querying children of any tile anymore.